.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I don't get it, there is no husky.sh script so do not understand why calling it ... so i assume husky install do that ...

Reply:
Yes exactly, husky.sh is created with husky install